cra-device-class-aware-loading/server.js

```diff
@@ -0,0 +1,109 @@
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const cors = require('cors');
+const app = express();
+const DeviceApiWeb = require('deviceatlas-deviceapi').DeviceApiWeb;
+const stringSimilarity = require('string-similarity');
+
+app.disable('x-powered-by');
+app.use(cors());
+app.use(express.static(path.join(__dirname, 'build')));
+
+// DeviceAtlas server-side API
+const deviceApi = (function () {
+  const devApi = new DeviceApiWeb();
+  
+  try {
+    devApi.loadDataFromFile('./lib/device-atlas/device-data-file/55268_20190722.json');
+  } catch (ex) {
+    devApi.error = ex;
+    console.log(ex.message);
+  }
+  
+  return devApi;
+})();
```

Comment:
Should we be using.... 

```js
const deviceApi = (() => {
  const devApi = new DeviceApiWeb();

  try {
    devApi.loadDataFromFile('./lib/device-atlas/device-data-file/55268_20190722.json');
  } catch (ex) {
    devApi.error = ex;
    console.log(ex.message);
  }

  return devApi;
})();
```

Reply:
Yes, arrow function looks better. I also search for conventional functions in Express.js server file and converted them to arrow functions.